lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/mv/XwikiController.java

```diff
@@ -102,6 +104,17 @@ public synchronized void updateBridgeInterface (BridgeInterface bi){
 	@Override
 	public void notifyVerification(String verificationProcessId)
 			throws LpRestException {
+
+	    VerificationStatus status = this.bridge.getVerificationStatus(verificationProcessId);
+	    //TODO: show the status.getStatus() somewhere in the wiki for the verification with id verificationProcessId?
+	    
+	    if(!status.getStatus().equals(StatusType.COMPLETED)){
```

Comment:
Well, I think that if the import failed, nothing is created in the Learn PAd platform, so why we should save this result somewhere? In the case we would like to keep it, I would suggest to store is as a kind of "log information" rather than something to show.  However, I would agree with @woshilapin, let's keep the TODO here, in case we will deal it.